packages/front-end/components/Features/RuleModal/ExperimentRefFields.tsx

```diff
@@ -1,31 +1,42 @@
 import { useFormContext } from "react-hook-form";
-import { FeatureInterface } from "back-end/types/feature";
-import { FaExternalLinkAlt } from "react-icons/fa";
+import { FeatureInterface, FeatureRule } from "back-end/types/feature";
+import { FaClock, FaExternalLinkAlt } from "react-icons/fa";
```

Comment:
Nit: should we us PiClock since we are moving all icons over to that?

Reply:
sure, updated